internal/repository/repository.go

```diff
@@ -0,0 +1,60 @@
+package repository
+
+import (
+	"github.com/VTerenya/employees/internal"
+	"strconv"
+)
+
+type RepositoryHandler interface {
+	GetPositions() map[string]internal.Position
+	GetEmployees() map[string]internal.Employee
+	AddPosition(p internal.Position)
+	AddEmployee(e internal.Employee)
+}
+
+type Repository struct {
+	data *database
+}
+
+func NewRepository(data *database) *Repository {
+	return &Repository{
+		data,
+	}
+}
+
+func (t Repository) GetPositions() map[string]internal.Position {
+	return t.data.GetPosition()
+
+}
+
+func (t Repository) GetEmployees() map[string]internal.Employee {
+	return t.data.GetEmployees()
+}
+
+func (t Repository) AddPosition(p internal.Position) {
+	t.data.GetPosition()[strconv.Itoa(len(t.data.GetPosition())+1)] = p
```

Comment:
Why you don't use position ID to store a position entity?

Reply:
+